docs/modules/ROOT/pages/integration_with_other_tools.adoc

```diff
@@ -90,7 +90,7 @@ provides LightTable integration.
 
 === RubyMine / Intellij IDEA
 
-RuboCop support is https://www.jetbrains.com/help/idea/2017.1/rubocop.html[available] as of the 2017.1 releases.
```

Comment:
`https://www.jetbrains.com/help/idea/2017.1/rubocop.html` returns 404.
Unfortunately, the release notes for the `2017.1` version are not available on the page`https://www.jetbrains.com/idea/download/other.html`.
Please let me know if you think this link is not a suitable replacement.